### ISSUE TITLE
[BUG] Able to Join Hub, unable to Join Specific Server

### ISSUE THREAD

**Issue by subrand0m** [MCCTeam/Minecraft-Console-Client#2162]
### Prerequisites

- [X] I made sure I am running the latest [development build](https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts)
- [X] I tried to [look for similar issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one
- [X] I have set `debugmessages=true` in config to diagnose my issue
- [X] I have redacted session tokens and passwords before attaching screenshots

### Minecraft Version

1.17 and 1.19.2

### Console Client Version

1.19

### Expected Behavior

Typing /server sun9 should connect you to the Skyblock server on MChub.com

### Actual Behavior

Minecraft Console Client v1.19.2 - for MC 1.4.6 to 1.19.2 - Github.com/MCCTeam
[MCC] [Settings] Loading Settings from C:\*\MinecraftClient-windows\MinecraftClient.ini
Resolving mchub.com...
[MCC] [Translations] No translation file found for en. (Looked 'C:\*\MinecraftClient-windows\lang\mcc\en.ini'
[MCC] [Translations] No translation file found for system language (en). (Looked 'C:\*\MinecraftClient-windows\lang\mcc\en.ini'
Color test: Your terminal should display [0123456789ABCDEF]: [0123456789ABCDEF]

[SessionCache] Initializing FileSystemWatcher for file: C:\*\MinecraftClient-windows\SessionCache.ini 
Loading Minecraft profiles: launcher_profiles.json
Loading session cache from disk: SessionCache.ini
Loaded session: *redacted*
Session data has been successfully loaded from disk.
Access token expiration time is 9/7/2022 11:25:44 AM
Cached session is still valid for subrand0m.
Saving session cache to disk
[KeysCache] Initializing FileSystemWatcher for file: C:\*\MinecraftClient-windows\ProfileKeyCache.ini 
Loading profile key cache from disk: ProfileKeyCache.ini
Loaded profile key, it will be refresh at 9/8/2022 2:18:42 AM
Profile keys has been successfully loaded from disk.
Cached profile key is still valid for subrand0m.
Success. (session ID: *redacted*)
Performing request to pc.realms.minecraft.net
Loading Minecraft profiles: launcher_profiles.json
Loading session cache from disk: SessionCache.ini
Loaded session: *redacted*
> GET /worlds HTTP/1.1
> Cookie: sid=token:*redacted*;user=subrand0m;version=1.19.2
> Cache-Control: no-cache
> Pragma: no-cache
> Host: pc.realms.minecraft.net
> User-Agent: Java/1.6.0_27
> Accept-Charset: ISO-8859-1,UTF-8;q=0.7,*;q=0.7
> Connection: close
>
>

< HTTP/1.1 200 OK
< Date: Tue, 06 Sep 2022 18:26:10 GMT
< Content-Type: application/json
< Content-Length: 14
< Connection: close
<
< {"servers":[]}
Retrieving Server Info...
{"version":{"protocol":760,"name":"Arcim 1.19.2"},"players":{"online":2782,"max":2783,"sample":[]},"description":{"text":"             §6●§e●§6● §7§lMC§6§lHUB §7[§a1.8§7-§a1.19§7] §6●§e●§6●§r\n       §6➥ §b§lPRISON§7, §e§lSKYBLOCK§7, §a§lSURVIVAL§7!§r§r"},"favicon":"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAWF0lEQVR4Xu1bCXRV5bU+TDJYLOu1tWK1VVd9UiuDgFMVLUggQiZIwmRlEAhDQiDTvYEQRlGLTJkHEkLm5M7zPXe+mcMUJoEKljEBFH2vraL29ZXvrb3PvdckpK8t9BnfWu61vnXOPfv//7P39///yb3n2xGE7+yurEwQhEZBEHIEQZALgrC3e4O7sAGCIKgEQVgsCELf7s5vhX3y8cdo8HpR7/Hg0sWL+PSTT5BTWISegrt29SqudnQgNDQUly9dgsflonavdm8nCEL+R9eu4fKFi/C63WhtaYHX48GRw4fR0d5OfXocv1fs2tVrqPN40dzYhNOnTqPe68WOnTvQ2traPdA+Z06d4oQcdju0Wi0TUKXRd2/X2n6lHR6XGx6nKwB3pyOhtZnH730jAjxuD7xuD27c+AR6owlBQUFIXZuGdWnp0FhsHOiIJ0ZISVBiLjfcvmNyihzJyTLcc8893K79Sge8Hi+P529LBPsT74x+/fr1Pgkd7dfgdroDSEpKwoe/v4DEZBmys7NRXLyXg7x+/SNfQh4cO3oMR9uOos5bJxGQIkduUQmMRhOT6fF44fKR5HK60NjQGDj3gwl0e74FBFzp6DIrLc0tKK2uQdiMmYhdtRrJKTIKUuHxeDhhr7eOk3MSYW4P0tI3QL42DRs3bcan//EHTt7pcsPhdMHpT9btQVNTs5Q0jVNXj4b6BtTV1aF///69S8LlKx0cqB8UuMkqYvGyZViweAlWxq+BwWjkhCkxf3J2hxN2hwvrN25CkkyOxBQ5HH4f+6WjwyURwP0c0vh0JBKPHD6Cv95C7xJw8UpHl4D9cLg8CIuehYhZc3DuwkX22+xO2OwOiHYng67J16djTbIMIWHhPr/ks/n8eSVlyCstR07xfuQUl8Bmc0K0Odh/+Egbrly93rsEXLjcAavdyRApSacbokNKYtDgwfjxgw8yIRS0lSDa+Vy0O9B64CDSN2/F6iTeJoHECA6nm69NnxmF4PCZmBwSxiitroVoszOR9Q2NuHi5g9p9r9f+HHx48TInZhHtsNLsMBHS8cKVDlxq7+BkLaINJtEGs9UGs43ORXgbGrFt+06kbdwKjcHEPmpnoQSdLsjT0rFw2UrMnPsbTA4Jx+TQMJQolLDaHbA5nHC63XyPAQMG9N4qePAnD8Fsd8Io2mGy2WGl/UkPOnrgNTTi1LkPYbE7YLSKDIPv6Mca2XrI1m/CmfMXYbbZu7SJTZQjNim1S3JVZgtElxtHT5yExe7E2ctXkJC+ufcI+P6wYTyzJpoVjwdWlxt60QaD1QYjJWSzwyDa4PTWQ2e1MrQ+GEQRccmpWJGQgjMXLkJ0e2Dx97fZMWfBYqxI4O3R2Xhr+DFw0GAsXp3cvc03Z+GRUTCIdg5cdFPwdmitog9WaCwSOGGbnc/V9Fm08TFmTQqWrk5Bnz59oLRYoKbZdzihE20YOGgQYtbcRoAwMTQC02bPwyvTwhiPPP7EbW2+MTt8+jQna3a5obGKUFmst4ESU5otUPrPLVYozGbGotUpjB/+eDgUZgtUVhuUFhFqUWRSFsWn4Fv7Q4jsXMd1WNwenjmVKHISCosFNSYzasxmPtaaragxSddqzRYGnVebzJi/KkVCvAx7SktRa6G2Ur8h3xuKBWvkRMAL39iM/rN27MIlKMxWKK12KK02/NsPfogqo7kLKo0mBu1ZOtK1tG1vo9JgQuSSOLy+KgWvr5Jhe9E+VJksfJ3w0rQwvJG8sfeW9z9i71++ghoiQLRDbXdC6NMHlUYzyg0mCXojyvRGuI6eYALonK5nllegTG9AyILlmLd6HebEp2JnSSlenDgJpdzHhAkhkVhfxr8Wv712/fObqDaLqDBZUGW28r5t+N0HKNWbOBF/MnWnfse+Up0BFUYz3ikswo6SEkyYMRdRsTJExcrxXkEhohcuwn6NHvt1BgwYOBipJd9yAj7983+hwmRFudGMMoMJbyYm4ckXXsZ+vRElOgODyLj/4UfRr39/FGt02KfV492ifZg0bTrGTglD8MI4zFgpwz6DCZYjx1Ck1qFIK70niN1V8u1eAYc+vIAyo8WXsBEbc/IwPjgCGrcXJXojJ+s6/j6eC43mFVCo0THuf/BBFKi0eDooDBOi5iN8hRx7yafS8nVCnz59Men1mC4EHDh/CU0fnof39AdoOnceWm89+cf22h7RNTTBXN/ABBARm/MLMS54BnJrlVDYnVA53ejbty/GBkdIBKi1UqJqSlKD0UFheD7idcxJ/S3ylZou+N5992Hca5FdCDjecQ1FOgOKdUZU2518PnDIvb23SlT1TTDVN2KfwYhCjRZpmTkYMzUcoyZN50QrrTY89OgjGDM1gpe0fwXkq7TIU2owanIongmbg6W7q5CjVCNXoUa2Qh04Hx0UTv36+Qk4cvkqbw8ao0inR6nRjKcnh/QeAZqWAzDXN0Ll8kDVfABL09IxZuoMRr5GjxylBs+GzcMz4XPxs5HjUag3IFel5SSzFGqMDArD+JDZWJ6rRKZCHQD5CElvv0vJ3RIEYZYgCEOOX/sYCqcbFVYbFC4PlE43HnlqbO8RYDp0lGe5/uhxGA4fw7hXfo1xIbMxKigC6tZD0Bw8gqenRWHs9FkY8fIUlJqtyFVrObmMWhWeejUUIyeHYuryVORq9IHEOyNbqYHK7WEyibxCnYHPicQivRF9e/PdoKntOHLUWtS63DA0NiJ4aTImL4jHazEyPP7sK1iSImMyaPmPnBzBbYmADIUKu2uV+OWkMPzy1VCMD5sL7ZFjyFCqkalUI1+rx36LiBqHCyqPF1lKDbIUKiaDVla2UiKH/rL06mvyEqdXWrY0Iyot5m/LQ0TCVkxbLseoKTPwyJjn8cRLk/1LtO+eWhUYNUrsqiECQvHUq2EYOTkMcdve5URLLFYej1BkNPP4tFoIlDR93lFZjbEvTUDwMv6xNPDrx+Q3bO8ptCizioHlSssxbPUmLN1VioiELRScunNIFDwRsLtWhV21Kk58ZFA4Hnj8SSYpQ6FmYnb7QNsiQBoRoVDDcvoskjOysL2iGsExt/9a/EZN2XoIJ86e45nLVeuwJG0DXlsmx7TlaxESt56C+0vngCiBDKUGeToDaj1eXv6/fFXaBuRP2pXBBPhBq8BPGB2zVDqkZGRjV7UCAwYPQf97BvYuAcU2J/ZU10qB0lN7TyamLpXx0vQtzy4Bbq+ogqPtKLLUOk7qF7+exsn//LlfB9ol7srAln2l2OknolaFnb4VQ9fk2Tn49xcmYVzYvN5N3me8HJP2ZPHMrMvNR0y2Eov3VCF4Gb/OuhHIntTTrDzev+vz92JDYRFGB0fh2ahFeCZyYfdkkLA7A4m7M5GWV4iUzBwMuvdejJgwBX379rtta31n/08sXRCEx/5OrH18bTb40Nn+KgjC6W7X/s9tI0nV169dZ5AYGrtyZfflSoaOK+0gZffk8ROs5AqC8KXP1+/Lz2+yNthYV48vPruJz/74p9vGuPnZTZw6dRoHWw/g5mef48ubXwTaVKq1ONx6AKQwX/z9eVz6/Xkcam2F1+VCHcllbjeOtbXhqy+4z1ddR747Cyi1DXUNfNyyhf+8dTecOX0azQ2N3N5pd1CbF8mhEx04+8FZSRX2aYiUbOfO7+7YhYkTJ976+eOPB1Thax1XuY2r+QASEpNuU4f9ajOpx3RPOtLny5cu+78+/0sMjfUNLFn7lVlSfXsYmcVLAqm/PuWWXmYiKVkGl+trZZfw1Zd/Doxhb2hmEfUXTz4Jtc4gKcQuN86dPYfmYye5/+zZcwN9/QT5Qcm/f+JkF4L/+J9/6CnGOzIOqHPwcXGrug/eh9QZPwFUKNHWdpTbkKaXkiIL+Eg9bjvShhPH+RUZ24ZNWyGTp+Lhhx+G1mgMtG1uaobCaGYChg4deouT6zTjTIDbw/K5RIZPTXa6YDDwO8h/id3qrP4Sxo0b33159fnpT38GFxFFOr4P5KhU65C6dh1L2iSNk9JLEjjJ3f7O7+3YgbT16fj88y/xwPDhcDicrADX1zdwkcXadWnUlj/X1zeisaEJ9XX1aG05gObGZk7YRYqUD163F8VlFYhZ2vVlyp3aX5x2SZJm2F09afL9nxgxQpKtnT4Z28EPQaFcpYEsdR1cLg9L5H5fXV1DYAySxhOSZThz5izsJIwSHCSLe5EsX8vFFoMGDWJiaAx6UDY1NTEheftL8asJLyN3337kleyHk9o4nCitqkFReUX3OO/IvmRJ2+ZATm4eK7Pdv90RAcOGDWPFllYBaf90To5ylZa1f7/qa7M5OBFShv2dKXkC9SEQCVQP4PZ4kJgi+bzeevYRMY1NTXyfGbPnYGpYOGP6jJmYNe833LczukR5h/aF1WZjhbe8Vokf3X9/TwQ8QjNEbSh5Cs7mkAioMdmQkCxnn+hTdQ8dPoLWg50JkCMpJZV9Lrebj1aWwJ1MXqJMjvar1wOyuUSOl5OeFjET02dEYt6CRUhIkUvkswxv808Wfa+4K/vK6pOshw8fjpLqmp4IWE0FS9SG6gSaWls5CUEQ7smvUmJ1sgwWCopkcpsdbcdP4OSpU4ExVifJEZ8o46RFhwONzS3w1NXxeaJsHYMIpTGPHD0Km1MiIjRqFkIioxEePYeTT/CRKPrlepEJGN011H/eYBRtDJK//wYBVCgJM80yFTW1tLKELQjCTza+twdxCTJJQfbDZkfLwUOBMeKTUxGXKLUx+2Enqd2Jndm5yMjLx8EjbTjU1gaHxwOLw8GSe1jUHCxcuhxrUuRYkyTn1WCxUf2BPQBBEJ7vGuo/b6z4smQt2vh8iPQWlt/y+shg2D11sFCNkMMFg423wH2x8nSWv0kuJxWZYLI7WU3232DtlrcRlyQP+OleelGE3VuHpLSNiE9eCyNVhLS0wuRwMtFmaYWxhcyei8aTp9Bw4iST5yeazgNZ3IVBK4oMkqxJCfZL3PSZAtLb7NBZRQmiLXBOX4Q2ZxYgav6igM/mreOjo+HrvwJvLItDvCwN1//0Gd9H46sfuHTjU6xMkmPdW+9AbbLAUUc1BjQ+EWTDofdP4fofP8OmbW8jp7QML0x4me+jpyILWrV2XgF3Z98f9n2orJLMrRZtUFlElrc1NEt2Bx9JGe4J1L/Y6sXkiEicu3odWtIPWTIX8UHHta+3QPpmxCTKcfGTT6XxLCSzi2g7ew7LE2RI3rAVI0ePhtVDKrTVByJKqiWgY05pKd7avj1Qk0AEPT1u3N0TMPuN+az+1poskgpsIQmcCLBDSZ//FzAB9ia8Gh4Fa0NDF9/Zax8HgluSIMei1TKc++gGFCaqJZAIP3WlHYvXyLA0Uc7bTWt3SDUEnWoPaFKIND63ijC53NKEWaX737UVVFWztl/tg8IiwuKt92n+ZiZG8ptRbZRQQzBZOIBskxeTI+fig49u+GoEpHFOtn+9AubHSzUCp69dRxXL5ybUcA2BhYlZLt/AzxiS3/1jEBRUaOEjQudwou7gQSbIT7LvGXV3lldVzZq+gvaw04Vqs4XlbypWokTovMJoQgVr+lQLYJbOjWa++S6jF6HzY3C8vUPqZzLzWKO/Xp73zF2VgnlxKXj/2kfct8Jg5HZUK/DGKhliN7zNBFDS/poDkuRpMihxncMFNX1vqG/kCaHJIbJ+d/3G3ROwp7wC1RaRJW2Sv2utNpTrpYRrrHTdwvI3w0CQ9P9yg5FvvrnKytp/jdXOfvb5iiV8BPSNXJYAIuFYx3WU6g0soUttjRj/ShCWrN3K7akf9adYqB5h+74SvJVfgK35BdiWXwjvwUOoNlqYPJqcWov0HLor27W/DPsCMrcR1sYmbMrNQ9I77+CxJ57gYsgRo0ajREuBGxlEBpFF/ZOLtIiOk7N8vp/8eiMqTewLBDd94UqELV6Ftvar2K83SNBJx1+MfQGxW3f7CDBDaXcwyTSOv//zr4XjpZAZvORpQvzFGUSUv80d27a9RTDU1bMgWazRw+Ctx4CBg9Cnb19WZUjCHjBoMDblFTBRTJbeCL2njm8es7sU09+M52vFrOoaUGnmmQkEN3HWQrwSNR8HLrXzffygtpGxcsyKX8ftSW1myd13H3//54LD8auQmYhYGssqNUMnwd/mjo3kbmtTCyu9hOwaRU+D/vXdfSWosIgoIeYtIqqs0peQ6LW/xcTohZwMJUVjUJvXly0PjPPs9Gg8My0KzReuSKqvjwBKMjJ+HWaskF6v7+tEIinD1PVtYwteCI1GeMxqrHxrDxNEinEZrbhOq+SOLfHd96BwulCooeD1yKiq7qlqy7M+MxvVoh06Tx3KzFbW7ckREr8BL0a+wZ+pf4Fah720vH1bhGzs1BkYO3UmGs5fRp5Kx20IlGj4SjkiVnKlGMrNVqm2QKNlosYHz8DMuFRErVqL6FWpmLVKzsSQzz9hXaK8E4tJS4e99QAHlqvUYnclE3CuW7NhMes3Ip/KX6gqTLSj2PdGJnLdDrwYuQB5ai3yVFpu40el24vGi+1ouHAF9ReuIE8p1QxQ7YDWU4edZZWY9JvlCFoYxwTkKjUot4jsp/OxUyMwZf4KhC5NQNDrMfjwTzdZps8nAuleKu3dE7AuM5sJ2FxYxEULldJ3/P/uRoAwO3YVsmupsEGDQq0BNb7f4hHrdrA0TskTgQQah0CkFGgNyFPr2J/DPi3fjxKUZ2Tjxcj5eCn6TfTt35/l8Hzuo2c/jeUnrUCjh7mxCdbmFvZRHO+UlN09AVuKS7CtpJQHy6HaHal4id7PBywoJgVEQJZPvaXZ3evbf4OG3ocxr0UiPa8goOnTkSCtCD32m6xwHz4C16HDLLTuLK/EhvxCnvVnw+bimdA5uO9HD7AqTPenVVZkMPG20ni8qHW4eLZL6M+kxSbdx0dg5zjvyNbnFUCelYNatxvWllbkSvuqCwE//OljWLH1LdQ2HUCWUo1slYbrAHzuh0YFhWPZpi3IVJC4KflyVBoU6A0wNTWzrE7iarZaC1NTC1KzclluGzUlHGOCZ+Lp4Eg8NWk6a48qb12gPU2I/5y0Sf81ItLecoAJ7BznHdnWohLW5Sk4v2bfwzt3yDNzOBjyswKsUAduPua1aIycHM7aYZ5GjyKTGXm09LUGLogg1ZfaU98fDR/O4memzQEijoorRk+JwFMTQ7CnVolqpysQR+d7+T/TmETw2uy8u0+eLCUzC2/tK+1yox6YvTVh2nQ89OijgUB2VtV2brPqqcnhXC22oWBvQEn2H2lcOq5M34gVm7ci0yL9jKXk/SQMvHcoE7UhJxeb9hZDtisDxvfPoMjmDFSV+CeJxvqX/A4go6WYvCebByXQLPZAwJ/lWbnI1epRYhGRUasEfe7W5tZz0W9yX5LQ1+YUcJ0AE9BJWv/BA8MD/YLi1mP01Eh+hlA/akOSehpty+xcpOftxdZ9+/FeZQ1yaDX5yEzJyOp+7zs3JiCAbEb3ogeyJRs2Ifad32JeQhIefPQxDBoyBP/g/waf8RH6RnenIAhDfT4MvvdeSYbfnRlAckYmlqRv5Ekh8sjfw+R8Z9/Z37H/AajcAEPCVuYOAAAAAElFTkSuQmCC","modinfo":{"type":"FML","modList":[]}}
Server is running Forge without mods.
Server version : Arcim 1.19.2 (protocol v760)
Translations file loaded.
[MCC] Version is supported. Logging in...
[DEBUG] Crypto keys & hash generated.
[MCC] Checking Session...
Performing request to sessionserver.mojang.com
> POST /session/minecraft/join HTTP/1.1
> Host: sessionserver.mojang.com
> User-Agent: MCC/1.19.2
> Content-Type: application/json
> Content-Length: 522
> Connection: close
>
> {"accessToken":*redacted*}

< HTTP/1.1 204 No Content
< Connection: close
< Date: Tue, 06 Sep 2022 18:26:11 GMT
< x-minecraft-request-id: ab307c726aecaebf
< Cache-Control: no-store
< X-Cache: Miss from cloudfront
< Via: 1.1 946b9edb2009c5508a0fbbd636f95014.cloudfront.net (CloudFront)
< X-Amz-Cf-Pop: SFO5-P2
< X-Amz-Cf-Id: NfcJPDxmrHQAgMMJj0CI8tPSNyhiUApWTkXararSYvHPQsKDdKYTeg==
<
<
[MCC] Server was successfully joined. Type '/quit' to leave the server.
[DEBUG] chat command = server sun9
Attempting to send you to sun9...
> Unhandled exception. System.IO.InvalidDataException: Failed to process incoming packet of type JoinGame. (PacketID: 37, Protocol: 760, LoginPhase: False, InnerException: System.Collections.Generic.KeyNotFoundException).
 ---> System.Collections.Generic.KeyNotFoundException: The given key 'minecraft:bskyblock_world_6' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at MinecraftClient.Mapping.World.SetDimension(String name)
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket(Int32 packetID, Queue`1 packetData)
   --- End of inner exception stack trace ---
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.HandlePacket(Int32 packetID, Queue`1 packetData)
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.Update()
   at MinecraftClient.Protocol.Handlers.Protocol18Handler.Updater(Object o)
   at System.Threading.Thread.StartCallback()
PS C:\*\MinecraftClient-windows>

### Steps to Reproduce the bug

1. Launch MCC via Powershell (Or CMD)
2. Once Connected to MChub.com type /Server Sun9 

### Anythings that could help diagnosing the bug

```txt
# Minecraft Console Client v1.19.2
# Startup Config File

# New to Minecraft Console Client? See README and sample configuration files here:
# https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config

# Want to upgrade to a newer version? See https://github.com/MCCTeam/Minecraft-Console-Client/#download
# Some settings missing here after an upgrade? Try to delete this file and relaunch MCC to generate a new one

[Main]
# General settings
# Leave blank to prompt user on startup
# Use "-" as password for offline mode
login=
password=
serverip=mchub.com
type=microsoft                     # Account type. mojang or microsoft. Also affects interactive login in console.
method=mcc                         # Microsoft Account sign-in method. mcc OR browser

# Advanced settings
# Make sure you understand what each setting does before changing anything!
language=en_GB
consoletitle=%username%@%serverip% - Minecraft Console Client
internalcmdchar=slash              # Use 'none', 'slash' or 'backslash'
messagecooldown=1                  # Minimum delay in seconds between messages to avoid being kicked for spam.
botowners=Player1,Player2,Player3  # Name list or myfile.txt, one name per line. /!\ Server admins can impersonate owners!
mcversion=auto                     # Use 'auto' or '1.X.X' values. Allows to skip server info retrieval.
mcforge=auto                       # Use 'auto', 'false' or 'true'. Force-enabling only works for MC 1.13+.
brandinfo=mcc                      # Use 'mcc', 'vanilla', or 'none'. This is how MCC identifies itself to the server.
chatbotlogfile=                    # Leave empty for no logfile
privatemsgscmdname=tell            # Used by RemoteControl bot
showsystemmessages=true            # System messages for server ops
showxpbarmessages=true             # Messages displayed above xp bar, set this to false in case of xp bar spam
showchatlinks=true                 # Decode links embedded in chat messages and show them in console
showinventorylayout=true           # Show inventory layout as ASCII art in inventory command
terrainandmovements=false          # Uses more ram, cpu, bandwidth but allows you to move around
inventoryhandling=false            # Toggle inventory handling (beta)
entityhandling=false               # Toggle entity handling (beta)
sessioncache=disk                  # How to retain session tokens. Use 'none', 'memory' or 'disk'
profilekeycache=disk               # How to retain profile key. Use 'none', 'memory' or 'disk'
resolvesrvrecords=fast             # Use 'false', 'fast' (5s timeout), or 'true'. Required for joining some servers.
accountlist=accounts.txt           # See README > 'Servers and Accounts file' for more info about this file
serverlist=servers.txt             # See README > 'Servers and Accounts file' for more info about this file
playerheadicon=true                # Only works on Windows XP-8 or Windows 10 with old console
exitonfailure=false                # Disable pauses on error, for using MCC in non-interactive scripts
scriptcache=true                   # Cache compiled scripts for faster load on low-end devices
timestamps=false                   # Prepend timestamps to chat messages
autorespawn=false                  # Toggle auto respawn if client player was dead (make sure your spawn point is safe)
minecraftrealms=true               # Enable support for joining Minecraft Realms worlds
moveheadwhilewalking=true          # Enable head movement while walking to avoid anti-cheat triggers

[Signature]
# Chat settings (affects minecraft 1.19+)
login_with_secure_profile=true     # Microsoft accounts only. If disabled, will not be able to sign chat and join servers configured with "enforce-secure-profile=true"
sign_chat=true                     # Whether to sign the chat send from MCC
sign_message_in_command=true       # Whether to sign the messages contained in the commands sent by MCC. For example, the message in "/msg" and "/me"
mark_legally_signed_msg=false      # Use green  color block to mark chat with legitimate signatures
mark_modified_msg=true             # Use yellow color block to mark chat that have been modified by the server.
mark_illegally_signed_msg=true     # Use red    color block to mark chat without legitimate signature
mark_system_message=false          # Use gray   color block to mark system message (always without signature)
show_modified_chat=true            # Set to true to display messages modified by the server, false to display the original signed messages
show_illegal_signed_chat=true      # Whether to display chat and messages in commands without legal signatures

[Logging]
# Only affect the messages on console.
debugmessages=true                # Please enable this before submitting bug reports. Thanks!
chatmessages=true                  # Show server chat messages
warningmessages=true               # Show warning messages
errormessages=true                 # Show error messages
infomessages=true                  # Informative messages (i.e Most of the message from MCC)
#chatfilter=                       # Regex for filtering chat message
#debugfilter=                      # Regex for filtering debug message
filtermode=blacklist               # blacklist OR whitelist. Blacklist hide message match regex. Whitelist show message match regex
logtofile=false                    # Write log messages to file
logfile=console-log-%username%-%serverip%.txt    # Log file name
prependtimestamp=false             # Prepend timestamp to messages in log file
savecolorcodes=false               # Keep color codes in the saved text (§b)

[AppVars]
# yourvar=yourvalue
# can be used in some other fields as %yourvar%
# %username% and %serverip% are reserved variables.

[Proxy]
# Connect to a server via a proxy instead of connecting directly
# If Mojang session services are blocked on your network, set enabled=login to login using proxy but connect directly to the server
# If connecting to port 25565 (Minecraft) is blocked on your network, set enabled=true to login & connect using the proxy
# /!\ Make sure your server rules allow Proxies or VPNs before setting enabled=true, or you may face consequences!
enabled=false                      # Use 'false', 'true', or 'login' for login only
type=HTTP                          # Supported types: HTTP, SOCKS4, SOCKS4a, SOCKS5
server=0.0.0.0:0000                # Proxy server must allow HTTPS for login, and non-443 ports for playing
username=                          # Only required for password-protected proxies
password=                          # Only required for password-protected proxies

[ChatFormat]
# MCC does it best to detect chat messages, but some server have unusual chat formats
# When this happens, you'll need to configure chat format below, see README > 'Detecting chat messages'
# Do not forget to uncomment (remove '#') these settings if modifying them
builtins=true                      # MCC support for common message formats. Set "false" to avoid conflicts with custom formats.
# public=^<([a-zA-Z0-9_]+)> (.+)$
# private=^([a-zA-Z0-9_]+) whispers to you: (.+)$
# tprequest=^([a-zA-Z0-9_]+) has requested (?:to|that you) teleport to (?:you|them)\.$

[MCSettings]
# Settings below are sent to the server and only affect server-side things like your skin
enabled=true                       # If disabled, settings below are not sent to the server
locale=en_US                       # Use any language implemented in Minecraft
renderdistance=medium              # Use tiny, short, medium, far, or chunk amount [0 - 255]
difficulty=normal                  # MC 1.7- difficulty. peaceful, easy, normal, difficult
chatmode=enabled                   # Use 'enabled', 'commands', or 'disabled'. Allows to mute yourself...
chatcolors=true                    # Allows disabling chat colors server-side
main_hand=left                     # MC 1.9+ main hand. left or right
skin_cape=true
skin_hat=true
skin_jacket=false
skin_sleeve_left=false
skin_sleeve_right=false
skin_pants_left=false
skin_pants_right=false

# =============================== #
#  Minecraft Console Client Bots  #
# =============================== #

# Although Minecraft Console Client is only a Chat client by default, it can also do many more things
# Below you will find configuration sections to enable Bots that can automatically perform tasks for you
# BY ENABLING BOTS, YOU AGREE TO COMPLY WITH SERVER RULES, AND FACE CONSEQUENCES FROM SERVER STAFF IF YOU DON'T

[Alerts]
# Get alerted when specified words are detected in chat
# Useful for moderating your server or detecting when someone is talking to you
enabled=false
alertsfile=alerts.txt              # List of words/strings to alert you on, e.g. "Yourname"
excludesfile=alerts-exclude.txt    # List of words/strings to NOT alert you on, e.g. "<Yourname>"
beeponalert=true                   # Play a beep sound when a word is detected in addition to highlighting

[AntiAFK]
# Send a command on a regular basis to avoid automatic AFK disconnection
# /!\ Make sure your server rules do not forbid anti-AFK mechanisms!
enabled=false
delay=600 #10 = 1s
command=/ping

[AutoRelog]
# Automatically relog when disconnected by server, for example because the server is restating
# Put keywords/strings such as "Server is restarting" in kick messages file to relog when the message is seen
# /!\ Use ignorekickmessage=true at own risk! Server staff might not appreciate if you auto-relog on manual kicks
enabled=false
delay=10                           # use 10 for 10 seconds, 10-60 for a random delay between 10 and 60 seconds
retries=3                          # retries when failing to relog to the server. use -1 for unlimited retries
kickmessagesfile=kickmessages.txt  # file with list of matches in kick messages that will trigger autorelog
ignorekickmessage=false            # when set to true, autorelog will reconnect regardless of kick messages

[ChatLog]
# Logs chat messages in a file on disk
enabled=false
timestamps=true
filter=messages
logfile=chatlog-%username%-%serverip%.txt

[Hangman]
# A small game to demonstrate chat interactions. Players can guess mystery words one letter at a time.
# You need to have ChatFormat working correctly and add yourself in botowners to start the game with /tell <bot username> start
# /!\ This bot may get a bit spammy if many players are interacting with it
enabled=false
english=true
wordsfile=hangman-en.txt
fichiermots=hangman-fr.txt

[ScriptScheduler]
# Schedule commands and scripts to launch on various events such as server join, date/time or time interval
# See README > 'Using the Script Scheduler' for more info
enabled=false
tasksfile=tasks.ini

[RemoteControl]
# Send MCC console commands to your bot through server PMs (/tell)
# You need to have ChatFormat working correctly and add yourself in botowners to use the bot
# /!\ Server admins can spoof PMs (/tellraw, /nick) so enable RemoteControl only if you trust server admins
enabled=false
autotpaccept=true
tpaccepteveryone=false

[AutoRespond]
# Run commands or send messages automatically when a specified pattern is detected in chat
# /!\ Server admins can spoof chat messages (/nick, /tellraw) so keep this in mind when implementing AutoRespond rules
# /!\ This bot may get spammy depending on your rules, although the global messagecooldown setting can help you avoiding accidental spam
enabled=false
matchesfile=matches.ini

[AutoAttack]
# Automatically attack hostile mobs around you
# You need to enable Entity Handling to use this bot
# /!\ Make sure server rules allow your planned use of AutoAttack
# /!\ SERVER PLUGINS may consider AutoAttack to be a CHEAT MOD and TAKE ACTION AGAINST YOUR ACCOUNT so DOUBLE CHECK WITH SERVER RULES!
enabled=false
mode=single                        # single or multi. single target one mob per attack. multi target all mobs in range per attack
priority=distance                  # health or distance. Only needed when using single mode
cooldownseconds=auto               # How long to wait between each attack. Use auto to let MCC calculate it
interaction=Attack                 # Possible values: Interact, Attack (default), InteractAt (Interact and Attack)

[AutoFishing]
# Automatically catch fish using a fishing rod
# You need to enable Entity Handling to use this bot
# You should also enable Inventory Handling to keep track of the fishing rod, although not mandatory
# /!\ Make sure server rules allow automated farming before using this bot
enabled=false
antidespawn=false

[AutoEat]
# Automatically eat food when your Hunger value is low
# You need to enable Inventory Handling to use this bot
enabled=false
threshold=6

[AutoCraft]
# Automatically craft items in your inventory
# See README > 'Using the AutoCraft bot' for how to use
# You need to enable Inventory Handling to use this bot
# You should also enable Terrain and Movements if you need to use a crafting table
enabled=false
configfile=autocraft\config.ini

[Mailer]
# Relay messages between players and servers, like a mail plugin
# This bot can store messages when the recipients are offline, and send them when they join the server
# /!\ Server admins can spoof PMs (/tellraw, /nick) so enable this bot only if you trust server admins
enabled=false
database=MailerDatabase.ini
ignorelist=MailerIgnoreList.ini
publicinteractions=false
maxmailsperplayer=10
maxdatabasesize=10000
retentiondays=30

[AutoDrop]
# Automatically drop items in inventory
# You need to enable Inventory Handling to use this bot
enabled=false
mode=include                      # include, exclude or everything. Include: drop item IN the list. Exclude: drop item NOT IN the list
items=                            # separate each item name with a comma ',': ItemOne,ItemTwo
# See this file for an up-to-date list of item types you can use with this bot:
# https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Inventory/ItemType.cs 

[ReplayMod]
# Enable recording of the game (/replay start) and replay it later using the Replay Mod (https://www.replaymod.com/)
# Please note that due to technical limitations, the client player (you) will not be shown in the replay file
# /!\ You SHOULD use /replay stop or exit the program gracefully with /quit OR THE REPLAY FILE MAY GET CORRUPT!
enabled=false
backupinterval=300                # How long should replay file be auto-saved, in seconds. Use -1 to disable
```

### Device

Desktop

### Operating System

Windows

### Server Address (If applicable)

mchub.com

**Comment by milutinke** [comment]
Duplicate of #2159
